### PR TITLE
Add backup backing image table

### PR DESCRIPTION
**Pull request by a110605**
### What this PR does / why we need it

- [x] Add a backupBackingImage table in backing image page
- [x] Backup backing image allows two operations : delete and restore 
- [x] Add `Backup` action for each backing image to do backup action
- [x] Add bulk `Delete` action on top of backup backing image table
- [x] Add bulk `Backup` action on top of backing image table
- [x] Add Search box for backup backing image table
- [x] Allow input secret / secretNamespace when restoring bbi

Other minor changes
- Add BackupBackingImages websocket icon on the right bottom corner
- Refactor duplicated logic to single one function
- Fix typo
 
### Issue

[[FEATURE] Add BackupBackingImage UI](https://github.com/longhorn/longhorn/issues/7541)
[[UI][FEATURE] Add backing image encryption and clone support](https://github.com/longhorn/longhorn/issues/8789)

### Test Result

**Backup BI and restore BBI**

https://github.com/longhorn/longhorn-ui/assets/5744158/58e74eb8-197d-40db-be15-421a7c0c72fb

**Search BI and BBI tables**

https://github.com/longhorn/longhorn-ui/assets/5744158/34ecab14-4d12-430e-95ca-55ffffd6fb2e

**Bulk backup BI**

https://github.com/longhorn/longhorn-ui/assets/5744158/f26a8f47-b9e6-43dd-96b7-b0049f37a31f

### Additional documentation or context

Try this PR using `a110605/longhorn-ui:0709issue7541amd64`

### CODE REVIEW

File: .stylelintrc [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "unit-no-unknown": null,
     "no-descending-specificity": null,
     "value-list-max-empty-lines": null,
-    "selector-type-no-unknown": null
+    "selector-type-no-unknown": null,
+    "selector-class-pattern": null
```
Comment (a110605) [inline .stylelintrc:26]: CodeFactor always complains the classname or id need to be `kebab-case` which is quite annoying. Disable this rule.
Screenshot 2024-07-01 at 5 11 17 PM

**Review by derekbit — APPROVED** [review]
LGTM

**Comment by derekbit** [comment]
@ChanYiLin Can you help review it? Thank you.

**Comment by ChanYiLin** [comment]
from feature perspective, LGTM.

**Review by ChanYiLin — APPROVED** [review]

**Pull request merged**